inc/common.class.php

```diff
@@ -621,9 +630,9 @@ public static function processMassiveActionsForOneItemtype(
                 }
                 $_SESSION['plugin_pdf']['type']   = $item->getType();
                 $_SESSION['plugin_pdf']['tab_id'] = serialize($tab_id);
-                $webDir                           = Plugin::getWebDir('pdf');
+                $webDir                           = Plugin::getPhpDir('pdf', false);
                 echo "<script type='text/javascript'>
-                      location.href='$webDir/front/export.massive.php'</script>";
+                      location.href='/plugins/pdf/front/export.massive.php'</script>";
```

Comment:
PDF export is broken, both for individual items and from the item list.

![image](https://github.com/user-attachments/assets/bda95404-30b1-42ee-84d7-72b83f0ca2fa)